### ISSUE TITLE
Remove IO From Event Loop

### ISSUE THREAD

**Issue by magico13** [magico13/ha-emporia-vue#2]
I'm logging in and getting the device data to set up the sensors apparently within the event loop per the logs. I need to figure out where to move that logic so I can still set up the sensors correctly.

Example:
```
2020-05-20 23:16:13 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for emporia_vue doing I/O at custom_components/emporia_vue/__init__.py, line 61: result = vue.login(username=email, password=password)
```

**Comment by haddood** [comment]
Logger: homeassistant.util.async_
Source: util/async_.py:120
First occurred: 6:56:47 PM (9 occurrences)
Last logged: 6:57:14 PM

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for emporia_vue doing I/O at custom_components/emporia_vue/__init__.py, line 61: result = vue.login(username=email, password=password)
Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for emporia_vue doing I/O at custom_components/emporia_vue/sensor.py, line 15: vue_devices = vue.get_devices()
Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for emporia_vue doing I/O at custom_components/emporia_vue/sensor.py, line 20: device = vue.populate_device_properties(device)

**Comment by alex-savin** [comment]
Still having that issue. HA v0.114.3
```
Log Details (WARNING)
Logger: homeassistant.util.async_
Source: util/async_.py:105
First occurred: 5:28:06 PM (9 occurrences)
Last logged: 5:28:08 PM

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for emporia_vue doing I/O at custom_components/emporia_vue/__init__.py, line 61: result = vue.login(username=email, password=password)
Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for emporia_vue doing I/O at custom_components/emporia_vue/sensor.py, line 15: vue_devices = vue.get_devices()
Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for emporia_vue doing I/O at custom_components/emporia_vue/sensor.py, line 20: device = vue.populate_device_properties(device)
```

**Comment by magico13** [comment]
I'll post in this issue when I have the work for that complete. I haven't been doing much work with this library as of late but this is one of my priorities when my focus comes back to this project.

**Comment by error420** [comment]
2020-11-09 08:54:48 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for emporia_vue doing I/O at custom_components/emporia_vue/__init__.py, line 61: result = vue.login(username=email, password=password)

**Comment by magico13** [comment]
Fixed in v0.2.0
